src/js/disability-benefits/526EZ/config/form.js

```diff
@@ -78,593 +87,662 @@ const formConfig = {
-                      };
-                    },
-                    attachmentSchema: {
-                      'ui:title': 'Document type'
-                    },
-                    attachmentName: {
-                      'ui:title': 'Document name'
-                    }
-                  }),
-                  { 'ui:description': additionalDocumentDescription }
-                )
-              }
-            }
           },
           schema: {
             type: 'object',
             properties: {
-              disabilities: {
-                type: 'array',
-                items: {
-                  type: 'object',
-                  properties: {
-                    additionalDocuments: {
-                      type: 'array',
-                      items: {
-                        type: 'object',
-                        required: ['name', 'attachmentId'],
-                        properties: {
-                          name: {
-                            type: 'string'
-                          },
-                          size: {
-                            type: 'integer'
-                          },
-                          confirmationCode: {
-                            type: 'string'
-                          },
-                          attachmentId: {
-                            type: 'string',
-                            'enum': [
-                              '1',
-                              '2',
-                              '3',
-                              // '4',
-                              '5',
-                              '6'
-                            ],
-                            enumNames: [
-                              'Discharge',
-                              'Marriage related',
-                              'Dependent related',
-                              // 'VA preneed form',
-                              'Letter',
-                              'Other'
-                            ]
-                          }
-                        }
-                      }
-                    }
-                  }
-                }
-              }
+              servicePeriods
             }
           }
-        },
-        evidenceSummary: {
-          title: 'Summary of evidence',
-          path: 'supporting-evidence/:index/evidence-summary',
-          showPagePerItem: true,
-          arrayPath: 'disabilities',
-          uiSchema: {
-            disabilities: {
-              items: {
-                'ui:title': 'Summary of evidence',
-                'ui:description': evidenceSummaryView
-              }
-            }
-          },
-          schema: {
-            type: 'object',
-            properties: {
-              disabilities: {
-                type: 'array',
-                items: {
-                  type: 'object',
-                  properties: {}
-                }
-              }
+        }
+      },
+      chapterTwo: {
```

Comment:
I suppose it probably doesn't matter, but this isn't the right level for these chapters; these are still nested in the `reviewVeteransDetails` chapter.